explorer/client.properties

```diff
@@ -1,19 +1,18 @@
 # Optional. A hostname of the Server. Use localhost by default if not specified.
-scalar.ledger.client.server_host=localhost
```

Comment:
I think we can remove this because a user surely has done the execution with his client.properties and has his own.

Reply:
Deleted!